docs/registry.md

```diff
@@ -71,5 +72,63 @@ If the module is successfully registered, you can use this converter through con
 
 ```python
 converter_cfg = dict(type='Converter1', a=a_value, b=b_value)
-converter = build_converter(converter_cfg)
+converter = CONVERTERS.build(converter_cfg)
+```
+
+## Hierarchy Registry
+
+Hierarchy structure is used for similar registries from different packages.
+For example, both [MMDetection](https://github.com/open-mmlab/mmdetection) and [MMClassification](https://github.com/open-mmlab/mmclassification) have `MODEL` registry define as followed:
+In MMDetection:
+
+```python
+from mmcv.utils import Registry
+from mmcv.cnn import MODELS as MMCV_MODELS
+MODELS = Registry('model', parent=MMCV_MODELS)
+
+@MODELS.register_module()
+class NetA(nn.Module):
+    def forward(self, x):
+        return x
+```
+
+In MMClassification:
+
+```python
+from mmcv.utils import Registry
+from mmcv.cnn import MODELS as MMCV_MODELS
+MODELS = Registry('model', parent=MMCV_MODELS)
+
+@MODELS.register_module()
+class NetB(nn.Module):
+    def forward(self, x):
+        return x + 1
+```
+
+We could build two net in either MMDetection or MMClassification by:
+
+```python
+import mmcls # import mmcls to register models
```

Comment:
Could this be omitted? It is unexpected that users need to modify the source codes apart from setting the configs.